apps/charterafrica/src/utils/fetchJson.js

```diff
@@ -1,5 +1,14 @@
 import { stringify } from "qs";
 
+export class FetchError extends Error {
+  constructor(message = "Something went wrong", data = {}, status = 500) {
+    super();
+    this.message = message;
```

Comment:
```suggestion
  constructor(message = "Something went wrong", data = {}, status = 500) {
    super(message);
```
`message` should be handled by [`super`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Error), no?